nvim-oxi/src/api/global.rs

```diff
@@ -116,7 +116,15 @@ pub fn get_mode() -> Dictionary {
 
 // get_runtime_file
 
-// get_var
+/// Binding to `nvim_get_var`.
+pub fn get_var<Value>(name: &str) -> Result<Value>
+where
+    Value: FromObject,
+{
+    let mut err = NvimError::new();
+    let obj = unsafe { nvim_get_var(name.into(), &mut err) };
+    err.into_err_or_flatten(|| Value::from_obj(obj))
```

Comment:
Just a nitpick: I prefer using `foo.into()` as opposed to `Foo::from(foo)` whenever the compiler can infer the type, as it reads from left to right like natural language. Closures are an exception, where `.map(Foo::from)` > `.map(|foo| foo.into())`.

Reply:
I'm sorry, but I don't quite get which part of the code you would suggest to change 😅

Reply:
Oops my bad, I read that as `Value::from` (from the `From` trait) instead of `Value::from_obj`.